docs/layouts/shortcodes/guides/tools.md

```diff
@@ -1,37 +1,11 @@
-Next, you need a couple tools to interact with your Platform.sh project, one of which you likely already have.
+Next, you need a couple tools to interact with your project, one of which you likely already have.
 
-* [Git](https://git-scm.com/).
+* [Git](https://git-scm.com/downloads).
   Git is the primary tool you use to manage everything your app needs to run.
   Every commit pushed results in a new deployment, and all of your configuration is driven almost entirely by a small number of YAML files in your Git repository (which we will get to in the steps below).
   Your infrastructure, described in these files, becomes part of your application itself - completely transparent and version-controlled.
   If you do not already have Git on your computer, you should [install it now](https://docs.github.com/en/get-started/quickstart/set-up-git).
```

Comment:
:clipboard: Again, this shouldn't be the page we're linking to. The link for downloading is at the start. Not sure this sentence is necessary at all, but if so, not with this link.